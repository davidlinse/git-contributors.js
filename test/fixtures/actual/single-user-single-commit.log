John Doe john@doe.org
 17 files changed, 7 insertion(+), 6 deletion(-)
