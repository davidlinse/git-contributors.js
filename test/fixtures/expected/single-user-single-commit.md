+ John Doe <john@doe.org>  (1 - 100%)
