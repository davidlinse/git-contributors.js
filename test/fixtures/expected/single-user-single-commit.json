[
  {
    "commits": 1,
    "email": "john@doe.org",
    "name": "John Doe",
    "percent": 100
  }
]
